projects/iio_demo/src/app/main.c

```diff
@@ -45,7 +45,7 @@
 #include "iio_demo_dev.h"
 #include "app_config.h"
 #include "parameters.h"
-#include "error.h"
+#include <errno.h>
```

Comment:
errno is included in error.h

Reply:
there are troubles with SUCCESS being defined multiple times and wanted to avoid them